beaconclient/multi_beacon_client.go

```diff
@@ -80,6 +94,18 @@ func NewMultiBeaconClient(log *logrus.Entry, beaconInstances []IBeaconInstance)
 		client.ffAllowSyncingBeaconNode = true
 	}
 
+	if os.Getenv("BROADCAST_VALIDATION") != "" {
```

Comment:
document!

also don't call `os.Getenv` twice, rather once and then work on the string